src/executor/explain_physical_plan.cpp

```diff
@@ -1671,6 +1706,52 @@ void ExplainPhysicalPlan::Explain(const PhysicalMergeTop *merge_top_node,
     }
     explain_header_str += "(" + std::to_string(merge_top_node->node_id()) + ")";
     result->emplace_back(MakeShared<String>(explain_header_str));
+
+    {
+        String sort_expression_str = String(intent_size, ' ') + " - sort expressions: [";
+        auto &sort_expressions = merge_top_node->GetSortExpressions();
+        SizeT order_by_count = sort_expressions.size();
+        if (order_by_count == 0) {
+            UnrecoverableError("MERGE TOP without any sort expression.");
+        }
+        auto &order_by_types = merge_top_node->GetOrderbyTypes();
+        for (SizeT idx = 0; idx < order_by_count - 1; ++idx) {
+            ExplainLogicalPlan::Explain(sort_expressions[idx].get(), sort_expression_str);
+            sort_expression_str += " " + SelectStatement::ToString(order_by_types[idx]) + ", ";
+        }
+        ExplainLogicalPlan::Explain(sort_expressions.back().get(), sort_expression_str);
+        sort_expression_str += " " + SelectStatement::ToString(order_by_types.back()) + "]";
+        result->emplace_back(MakeShared<String>(sort_expression_str));
+    }
+
+    {
+        auto limit = merge_top_node->GetLimit();
+        static_assert(std::is_same_v<decltype(limit), u32>);
+        auto offset = merge_top_node->GetOffset();
+        static_assert(std::is_same_v<decltype(offset), u32>);
+        auto limit_after_offset = limit - offset;
```

Comment:
both limit and offset is u32, the limit_after_offset is also u32, which won't be less than zero.